lint/cadence_v1_analyzer.go

```diff
@@ -283,6 +283,9 @@ func (v *cadenceV1Analyzer) inspectTypeAnnotations(f func(typeAnnotation *ast.Ty
 
 	// Helper function to process a parameter list
 	processParameterList := func(parameterList *ast.ParameterList) {
+		if parameterList == nil {
```

Comment:
I was at first surprised this could happen for normal function declarations and expressions, but this function is also called for other cases, like transaction declarations where this might be the case.

Maybe leave a comment explaining why this is needed.

Please also add a test case

Reply:
Alright sounds good 👍